toothpick-runtime/src/main/java/toothpick/ScopeImpl.java

```diff
@@ -174,20 +171,26 @@ public String toString() {
     return builder.toString();
   }
 
-  private void installModule(Module module) {
+  private void installModules(boolean test, Module... modules) {
+    for (Module module : modules) {
+      installModule(module, test);
+    }
+  }
+
+  private void installModule(Module module, boolean test) {
```

Comment:
The boolean is dancing in position, last, first. Make it more homogenous please. First will be your only choice, because of the varargs in the other method, but it makes sense for what we add.